src/main/java/com/squareup/javapoet/AnnotationSpec.java

```diff
@@ -122,6 +122,9 @@ public static Builder builder(Class<?> type) {
   public Builder toBuilder() {
     Builder builder = new Builder(type);
     builder.members.putAll(members);
+    for (Map.Entry<String, List<CodeBlock>> entry : builder.members.entrySet()) {
```

Comment:
Sure. You're very welcome to change that.

Btw. all other toBuilder() method in *Spec classes should work - as there're no nested immutable collections. I think. Will write some unit tests asserting that.